src/components/apiComponents/exampleSelector.jsx

```diff
@@ -18,24 +21,37 @@ const ApiExampleSelector = ({
       <div className="relative flex w-full gap-2">
         <div className="relative flex h-12 w-full">
           <Select
-            onChange={(e) => handleSetExample(e.target.value)}
-            value={selectedExample}
+            onChange={(e) => {
+              const selectedName = e.target.value;
+              const index = examples.findIndex(example => example.name === selectedName);
+              handleSetExample(index);
+            }}
+            value={selectedExample === -1 ? '' : examples[selectedExample]?.name || ''}
             className={clsx(
               'dark:white block w-full appearance-none rounded-lg border border-black/10 bg-white/5 px-3 py-1.5 text-sm/6 text-black dark:border-white/15 dark:bg-transparent',
               'focus:outline-none data-[focus]:outline-2 data-[focus]:-outline-offset-2 data-[focus]:outline-white/25',
               '*:text-black dark:text-white'
             )}
           >
-            <option value={-1}>-</option>
-            <optgroup label="Solana Mainnet">
-              {examples.map((example, index) => {
-                return (
-                  <option key={index} value={index}>
+            <option value="">-</option>
+            {mainnetExamples.length > 0 && (
```

Comment:
Since devnet is the default url I feel like the devnet examples should be at the top and mainnet at the bottom?
Might make it less confusing since you can just pick the first from the list?

Reply:
+1